car-park/CarPark_init.cs

```diff
@@ -25,12 +48,12 @@
 );
 
 downaction():=(
+regional(hotlist,rest);
 hotlist=select(1..length(alldata),ishot(mouse().xy,alldata_#));
 moving=false;
 if(length(hotlist)==1,
-  sel=hotlist_1;
```

Comment:
@montaga @eric This is why global variables should be avoided. `sel` is used in two different contexts: For level selection (`int`) and for storing parts of the `hotlist` (`list`). During refactoring this hit me very hard. It took me about an hour to figure out what was going on (also because debugging is via `print`...). For new or refactored CindyScript code, the `regional` operator should be used where ever possible. It greatly reduces amount of spaghettis in our spaghetti code.